packages/application-components/src/components/page-content-containers/page-content-wide/page-content-wide.tsx

```diff
@@ -1,13 +1,13 @@
-// TODO: @redesign cleanup
 import { Children, type ReactNode } from 'react';
 import styled from '@emotion/styled';
-import { designTokens, useTheme } from '@commercetools-uikit/design-system';
+import { designTokens } from '@commercetools-uikit/design-system';
 import { useWarning } from '@commercetools-uikit/utils';
 
 export type TPageContentWide = {
   columns: '1' | '1/1' | '2/1';
   gapSize: '10' | '20';
   children: ReactNode;
+  // @deprecated
   themeParentSelector?: () => HTMLElement | null;
```

Comment:
This is mostly an internal prop we used in the MC frontend so I think it would be safe to remove it but I thought it would help to just deprecate it first.